lib/model/txproposal.js

```diff
@@ -71,6 +71,7 @@ TxProposal.create = function(opts) {
   x.excludeUnconfirmedUtxos = opts.excludeUnconfirmedUtxos;
   x.proposalSignaturePubKey = opts.proposalSignaturePubKey;
   x.proposalSignaturePubKeySig = opts.proposalSignaturePubKeySig;
+  x.derivationStrategy = opts.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP45;
```

Comment:
and in txp, both create and fromObj defaults to BIP45, to support old objects and old clients, right?